packages/management/tanzu-auth/bundle/.imgpkg__BACKUP/README.md

```diff
@@ -0,0 +1,104 @@
+# Tanzu Auth
```

Comment:
Question: Could you help provide some details that I could educate myself on what is `.imgpkg__BACKUP` used for ?

Reply:
accidental inclusion, ill delete it!